### DISCUSSION TITLE
PR #1660 broke inline data definition

### DISCUSSION THREAD

**Discussion by alexej-d** [alpinejs/alpine#1667] [7. Bugs (archived)] [answered]
I believe one of the latest PRs (namely #1660) broke simple data definitions like this `<div x-data="{ test: true }">` – this leads to an exception, saying data is undefined.

**Comment by danddanddand** [comment, marked as answer]
There is PR to fix it. Same issue as here (https://github.com/alpinejs/alpine/discussions/1664). Use 3.1.1 for now
